src/conduit/components/login.cljs

```diff
@@ -0,0 +1,19 @@
+(ns conduit.components.login
+  (:require [rum.core :as rum]
+            [conduit.components.header :refer [Header]]
+            [conduit.components.footer :refer [Footer]]
+            [conduit.components.forms :refer [LoginForm]]))
+
+(rum/defc Login [r route params]
+  [:div
+   (Header r route)
+   [:.auth-page
+    [:.container.page
+     [:.row
+      [:.col-md-6.offset-md-3.col-xs-12
+       [:h1.text-xs-center "Sign in"]
+       [:p.text-xs-center
+        [:a {:href "#/register"} "Need an account?"]]
+       (LoginForm r route params)
+       ]]]]
```

Comment:
Formatting.`]]]]` should be in the end of the previous line. Check [Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide) if you are not sure about any formatting rules.